Stripe/STPBankAccount.m

```diff
@@ -127,7 +124,7 @@ + (NSArray *)requiredFields {
              ];
 }
 
-+ (instancetype)decodedObjectFromAPIResponse:(NSDictionary *)response {
++ (nullable instancetype)decodedObjectFromAPIResponse:(nullable NSDictionary *)response {
```

Comment:
looking through the `requiredFields` list, should we add:
- account_holder_type
- fingerprint
- status
to support our nonnull annotation?